# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../frandom/frandom-1.1-udev-rules.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- a/10-frandom.rules	2013-05-21 15:46:58.523445033 +0200
+++ b/10-frandom.rules	2013-05-21 15:47:11.194306377 +0200
@@ -5,4 +5,4 @@
 # 50-udev.rules (0600) is too restrictive.
 #
 
-KERNEL=="[ef]random", NAME="%k", MODE="444", OPTIONS="last_rule"
+KERNEL=="[ef]random", MODE="444", OPTIONS="last_rule"
